pytorch_lightning/profiler/pytorch.py

```diff
@@ -69,7 +69,8 @@ def __init__(self, model: nn.Module) -> None:
         self._handles: Dict[str, List["RemovableHandle"]] = {}
 
     def _start_recording_forward(self, _: nn.Module, input: Tensor, record_name: str) -> Tensor:
-        record = record_function(record_name)
+        # Add [pl][module] in name for pytorch profiler to recognize
```

Comment:
Won't this make the trace harder to read?

Reply:
we will organize pytorch-lightning tags in a separate view and show without this header.

Reply:
this is confusing that the action name being profiled isn't the exact same as what shows up in the trace. could you describe how this is intended to be post-processed/viewed?

would it make sense for `record_function` to accept metadata that allows the profiler to post-process these records?

Reply:
Currently the `record_function` doesn't support metadata. What I mean is we will set up a separate view for pytorch-lightning in pytroch profiler (maybe called pytorch-lightning view) which shows the action name without pre-tags. The pytorch-lightning view is for user to easily understand profiling info in pytorch-lightning. In the older trace view the showed name is still exactly same as the action name.